From 328c6b3ac9408b3ad1169b6e50b53d4688cb9d72 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matt=C3=A9o=20Rossillol=E2=80=91=E2=80=91Laruelle?=
 <beatussum@protonmail.com>
Date: Fri, 24 Jul 2020 13:55:44 +0200
Subject: [PATCH] fix `--with*-<pkg>`
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

without this fix, some `--without-<pkg>` are interpreted as `--with-<pkg>`
and vice versa.

Signed-off-by: Mattéo Rossillol‑‑Laruelle <beatussum@protonmail.com>
---
 configure.ac | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/configure.ac b/configure.ac
index d98e4dd..1038509 100644
--- a/configure.ac
+++ b/configure.ac
@@ -82,9 +82,7 @@ AX_CHECK_COMPILE_FLAG([[-std=c++11]],,
 
 # use GPM (General Purpose Mouse)
 AC_ARG_WITH([gpm],
-            [AS_HELP_STRING([--without-gpm], [Disable GPM mouse support])],
-            [with_gpm=no],
-            [with_gpm=yes])
+            [AS_HELP_STRING([--without-gpm], [Disable GPM mouse support])])
 if test "x$with_gpm" = "xyes"
 then
   AC_CHECK_LIB([gpm],
@@ -107,9 +105,7 @@ fi
 
 # unit test
 AC_ARG_WITH([unit-test],
-            [AS_HELP_STRING([--with-unit-test], [build unit tests])],
- 	    [with_unit_test=yes],
-            [with_unit_test=no])
+            [AS_HELP_STRING([--with-unit-test], [build unit tests])])
 if test "x$with_unit_test" = "xyes"
 then
   AC_MSG_NOTICE(enabled cppunit test)
-- 
2.26.2

